openshift/installer
# Bug 2027501: bump RHCOS 4.10 bootimage metadata

These changes will update the RHCOS 4.10 bootimage metadata in the installer.
This change includes fixes for the following BZs:

Bug 1990506 - Missing udev rules in initramfs for /dev/disk/by-id/scsi-* symlinks

This change will also introduce artifacts for for Aliyun, AWS GovCloud regions, and Nutanix.

Changes generated with:
```
$ cosa shell
[coreos-assembler]$ plume cosa2stream --target data/data/coreos/rhcos.json --distro rhcos --no-signatures \
--url https://rhcos-redirector.apps.art.xq1c.p1.openshiftapps.com/art/storage/releases aarch64=410.84.202112060851-0 \
ppc64le=410.84.202112041603-0 s390x=410.84.202112040202-0  x86_64=410.84.202112040202-0
```
Verification Steps:

1. Install a new 4.10 cluster
2. `oc debug node/<node name> -- chroot /host rpm-ostree status`
3. Verify that the deployment version matches the version from this PR
   that matches the architecture you are testing on. (i.e. `aarch64`
   should have version `410.84.202112060851-0`)
